modules/service.sh

```diff
@@ -5,7 +5,7 @@ service_from_command() {
 
      echo "$command" | sed -n -r \
          's,^is-(.+)-enabled$,\1,p;
-          s,^(enable|disable)-(.+)$,\2,p'
+          s,^(enable|disable|update)-(.+)$,\2,p'
```

Comment:
We shouldn't add a global `update-<service>` command, otherwise all services would get it (and we'd have to add `<service>_update` functions for each of the others with an unsupported message.
I'd suggest adding a `update-fips` entry to the `case` in main, and having needed code there